maintainers/maintainer-list.nix

```diff
@@ -3900,6 +3900,11 @@
     github = "pakhfn";
     name = "Fedor Pakhomov";
   };
+  pamplemousse = {
+    email = "xav.maso@gmail.com";
+    github = "Pamplemousse";
```

Comment:
As per brand new policy if you set `github` you must now also set `githubid`. Your `githubid` is [2647236](https://api.github.com/users/pamplemousse).